test/e2e_node/resource_metrics_test.go

```diff
@@ -96,6 +96,10 @@ var _ = SIGDescribe("ResourceMetricsAPI", feature.ResourceMetrics, func() {
 					"": boundedSample(10*e2evolume.Mb, memoryLimit),
 				}),
 
+				"node_swap_usage_bytes": gstruct.MatchElements(nodeID, gstruct.IgnoreExtras, gstruct.Elements{
```

Comment:
We'd expect this to be 0 for these tests actually?

Reply:
Can you make sure that doesn’t break the general swap fedora or Ubuntu test lanes?

Reply:
I guess they should not be swapping but we should trigger the test anyway.

Reply:
@kannon92 done! all swap lanes have passed.